src/platforms/python/guides/django/index.mdx

```diff
@@ -122,6 +124,12 @@ You can pass the following keyword arguments to `DjangoIntegration()`:
 
   The default is `True`.
 
+- `cache_spans`:
+
+  Create spans and track performance of all read operations to configured caches. Set to `False` to disable. The spans also include information if the cache access was a hit or a miss.
```

Comment:
Small nit: would swap the second and third sentence to be consistent with the rest of the option descriptions:
```suggestion
  Create spans and track performance of all read operations to configured caches. The spans also include information if the cache access was a hit or a miss. Set to `False` to disable.
```